addons/point_of_sale/static/src/app/customer_display/customer_display_service.js

```diff
@@ -58,7 +58,7 @@ export class LocalDisplay extends Reactive {
         // animations.
         const { body: displayBody, head: displayHead } = this.popupWindow.document;
         const container = document.createElement("div");
-        container.innerHTML = await this.globalState.customerDisplayHTML(closeUI);
+        container.innerHTML = await this.pos.customerDisplayHTML(closeUI);
```

Comment:
Hi team @odoo/rd-security. This PR is ready to be merged but it requires your approval. The security ci is triggered because of this change. Can you approve?